style.css

```diff
@@ -1,3 +1,399 @@
+body {
+  /* background-color: #f7f2f2; */
+  background-color: #f9f9f9;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  margin: 0;
+}
+
+
+@media only screen and (min-width: 500px) {
+  body {
+    padding-bottom: 50px;
+  }
+}
+
+.header-primary {
+  display: grid;
+  grid-template-areas: 
+  "header-logo . hamburger-menu-display";
+  grid-template-columns: 3fr 1fr 30px;
+  padding: 5%;
+  background-color: #403d3d;
+  /* border: 2px solid red; */
+}
+
+.header-logo {
+  grid-area: header-logo;
+  font-family: Roboto, Cambria;
+  font-weight: bold;
+  font-size: 30px;
+  color: #bfd8fa;
+  /* border: 2px solid red; */
+}
+  
+.header-anchor-register, .header-anchor-login {
+display: none;
+}
+
+.hamburger-menu-display {
+  grid-area: hamburger-menu-display;
+}
+
+.nav-list-mobile {
+  list-style: none;
+}
+
+.nav-anchor-mobile {
+  text-decoration: none;
+  background-color: #0862d7;
+  color: white;
+}
+
+.bar {
+  width: 30px;
+  color: white;
+  border-bottom: 2px solid white;
+}
+
+.header-menu-mobile {
+  display: none;
+}
+
+.header-menu-mobile.open {
+  display: block;
+}
+
+@media only screen and (min-width: 500px) {
+  .header-primary {
+    display: grid;
+    grid-template-areas: 
+    "header-logo header-anchor-login header-anchor-register";
+    column-gap: 30px;
+    grid-template-columns: 3fr .2fr .3fr;
+    padding: 2% 5% 2% 2%;
+    background-color: #403d3d;
+  }
+
+  .header-anchor-login {
+    display: inline;
+    grid-area: header-anchor-login;
+    text-decoration: none;
+    font-size: 1.2em;
+    color: white;
+    padding-top: 10px;
+  }
+
+  .header-anchor-login:hover {
+    display: inline;
+    color: #0862d7;
+    text-decoration: underline;
+    text-transform: uppercase;
+  }
```

Comment:
Well done for adding hover state 👍
Try not to repeat anything that doesnt change eg `display: inline;`